scripts/zones/Apollyon/mobs/Jidra.lua

```diff
@@ -41,13 +41,13 @@ function onMobDespawn(mob)
 
 
  if (
-IsMobDead(16932882)==true and
-IsMobDead(16932883)==true and
-IsMobDead(16932884)==true and
-IsMobDead(16932885)==true and
-IsMobDead(16932886)==true and
-IsMobDead(16932887)==true and
-IsMobDead(16932888)==true
+limbus.isMobDead(16932882)==true and
```

Comment:
`==true` can be removed

Reply:
This is just from a mass function rename, still gotta go through all the mobs and fix styling and replace hard coded IDs